src/vue/pages/TradeExchange.vue

```diff
@@ -170,7 +170,7 @@ export default {
           quote_asset: this.assetPair.quote,
           is_buy: false,
         })
-        this.sellOffersList = response.data
+        this.sellOffersList = this.sortOffersList(response.data, 'bids')
```

Comment:
we shouldn’t sort the items by ourself. please try passing `order` query param to request with `desc` (or `asc`) value to reach the result